packages/template-block/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const SETTING_ID = 'template';
```

Comment:
Be more explicit about the variable and what it represent

Reply:
Fixed.